### PR TITLE
Merging website code

### PR DESCRIPTION
**Pull request by LaurynFerkul**
- added form, display and image component 
- made an axios call to rebrickable
- styled the elements on the page and added media queries 
- started to handle errors

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,38 +1,351 @@
...
+        text-align: center;
+    }
+
+    .legoHeads {
+        max-width: 30%;
+    }
+}
+
+/****************** MEDIA QUERY FOR MOBILE START ******************/
+@media (max-width: 450px) {
```
Comment (DeveloperWeb123) [inline src/App.css:572]: I noticed that when I swap to mobile I can't really see the input options - maybe add an anchor of visual arrow pointing down so the user knows where they need to input something

File: src/DisplayLegoSets.js [hunk 2]
```diff
@@ -0,0 +1,40 @@
+import redLegoTop from "./assets/redLegoTop.svg";
+
+const displayLegoSets = (props) => {
+
+    return (
+        <section className="displaySets"> 
+            {
+            props.legoSets.length === 0 
+                ? (<h2>Sorry no lego sets could be found</h2>)
```
Comment (CourtG) [inline src/DisplayLegoSets.js:9]: nice error catch!

Comment (CourtG) [inline src/DisplayLegoSets.js]: nice error catching!

File: src/Form.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+        const { name, value } = event.target;
+
+        setInputValues({
+            ...inputValues,
+            [name]: value,
+        });
+
+    }
+
+    console.log(props.error)
```
Comment (CourtG) [inline src/Form.js]: I'm not sure if you could take out this log? I'm really just trying to find something to comment on

File: src/Form.js [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+                        value={inputValues.pieces}
+                        onChange={handleChange}
+                        type="text"
+                        id="pieces"
+                        className="pieces"
+                        name="pieces"
+                        placeholder="Number of pieces"
+                    />
+                </div>
+                {/* {props.error.length === 0
```
Comment (CourtG) [inline src/Form.js]: get rid of these comments for tomorrow morning maybe? *shrugs*

**Review by CourtG — COMMENTED** [review]
I always love your apps! SO pretty! functional, interesting, and something i think people can really use. can't wait to see what you do with it in the future!

File: src/App.js [hunk 5]
```diff
@@ -1,25 +1,78 @@
...
+
+
+  useEffect(() => {
+    // make an axios call to the Rebrickable Api to get back an array of lego sets
+
+    if (userInputValues  && userInputValues.length !== 0 && userInputValues.year !== "") {
+      axios({
+        baseURL: 'https://rebrickable.com',
+        url: '/api/v3/lego/sets/',
+        method: 'GET',
```
Comment (kianaomoerah) [inline src/App.js]: axis method is by default GET, if you want to shorten your code a tiny bit you can remove this line

File: src/App.js [hunk 6]
```diff
@@ -1,25 +1,78 @@
...
+          min_parts: userInputValues.pieces,
+        }
+      }).then((res) => {
+
+        // Store this array of lego sets in state
+        setLegoSets(res.data.results)
+        
+      }).catch(function (error) {
+        if (error.response){
+            alert("Please try a new input")
```
Comment (kianaomoerah) [inline src/App.js]: great error handling!

File: src/App.js [hunk 7]
```diff
@@ -1,25 +1,78 @@
...
+
+    setUserInputValues({
+      ...inputValue,
+      [name]: value,
+    });
+}
+
+return (
+  <>
+    <header>
```
Comment (kianaomoerah) [inline src/App.js:60]: for cleaner code you might want to consider making the header and footer a separate component :)

**Pull request merged**